src/diffusers/pipelines/latent_consistency_models/pipeline_latent_consistency_img2img.py

```diff
@@ -142,6 +142,7 @@ class LatentConsistencyModelImg2ImgPipeline(
         - [`~loaders.LoraLoaderMixin.load_lora_weights`] for loading LoRA weights
         - [`~loaders.LoraLoaderMixin.save_lora_weights`] for saving LoRA weights
         - [`~loaders.FromSingleFileMixin.from_single_file`] for loading `.ckpt` files
+        - [`~loaders.IPAdapterMixin.load_ip_adapter`] for loading IP Adapters
 
     Args:
         vae ([`AutoencoderKL`]):
```

Comment:
Should we also instantiate the `image_encoder` to None like it's done here?

https://github.com/huggingface/diffusers/blob/3b9b98656e2e3a106bdb2ec0fd22fa18af856075/src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py#L167

Reply:
It is instantiated to None already, right? Only the type hint is different from `pipeline_stable_diffusion.py` (it is `Optional[CLIPVisionModelWithProjection]` instead of `CLIPVisionModelWithProjection`).

Reply:
I don't see it here:
https://github.com/huggingface/diffusers/blob/08b453e3828f80027d881bb460716af95e192bcd/src/diffusers/pipelines/latent_consistency_models/pipeline_latent_consistency_img2img.py#L173

I don't see `image_encoder` in the constructor of the pipeline. What am I missing out on?

Reply:
I think you're looking at the wrong commit, Sayak :sweat_smile: Please check [here](https://github.com/huggingface/diffusers/pull/5900/files#diff-303eba33c2cea614b00e4533b90bc65f60023ea85eedcd112e4401837e8e6a96R183) and [here](https://github.com/huggingface/diffusers/pull/5900/files). The line instantiating image_encoder to None does indeed exist...

![image](https://github.com/huggingface/diffusers/assets/72266394/51e6a9d3-8b47-4f0a-b4b9-e1e3a96934a6)

Reply:
Indeed. I see it here:
https://github.com/a-r-r-o-w/diffusers/blob/0d64ce3d0a21014c682fe82b09e6bd392d6515f1/src/diffusers/pipelines/latent_consistency_models/pipeline_latent_consistency_img2img.py#L183